cmd/tendermint/commands/init.go

```diff
@@ -17,9 +19,11 @@ import (
 
 // InitFilesCmd initializes a fresh Tendermint Core instance.
 var InitFilesCmd = &cobra.Command{
-	Use:   "init",
-	Short: "Initialize Tendermint",
-	RunE:  initFiles,
+	Use:       "init [full|validator|seed]",
+	Short:     "Initializes a Tendermint node",
+	ValidArgs: []string{"full", "validator", "seed"},
+	Args:      cobra.MaximumNArgs(1),
```

Comment:
```suggestion
	Args:      cobra.ExactArgs(1),
```

Reply:
I thought about forcing users to enter a mode but instead I've made it so that you can continue to write just `tendermint init` but the command will then ask you to enter a mode before continuing. I thought it might be less of a jump to people who have been entering `tendermint init` their entire life